src/main/java/io/r2dbc/postgresql/client/TransactionStatus.java

```diff
@@ -50,10 +49,10 @@
     static TransactionStatus valueOf(ReadyForQuery.TransactionStatus t) {
         Objects.requireNonNull(t, "t must not be null");
 
-        return Arrays.stream(values())
-            .filter(type -> type.discriminator == t)
-            .findFirst()
-            .orElseThrow(() -> new IllegalArgumentException(String.format("%s is not a valid transaction status", t)));
+        if ( t == IDLE.discriminator ) return IDLE;
```

Comment:
My kingdom for an enumeration-aware `switch` statement.  Can you please add curly braces here? I know it's more verbose, but we're an always-curly braces project.

Reply:
Actually, scratch the previous.  Can you please match this to the style you used in `BackendMessageDecoder` instead?